modules/sbo-release-notes-1-1.adoc

```diff
@@ -0,0 +1,95 @@
+[id="sbo-release-notes-1-1_{context}"]
```

Comment:
```suggestion
// Module included in the following assembly:
//
// * applications/connecting_applications_to_services/sbo-release-notes.adoc
:_content-type: REFERENCE
[id="sbo-release-notes-1-1_{context}"]
```

Reply:
Only six of the release notes in /modules seem to have a content type attribute, but those that do have picked "REFERENCE" as the type. That seems reasonable to me.